Formula/percona-xtrabackup.rb

```diff
@@ -0,0 +1,77 @@
+class PerconaXtrabackup < Formula
+  desc "Open source hot backup tool for InnoDB and XtraDB databases"
+  homepage "https://www.percona.com/software/mysql-database/percona-xtrabackup"
+  url "https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.2/source/tarball/percona-xtrabackup-2.4.2.tar.gz"
+  sha256 "faeac6f1db4a1270e5263e48c8a94cc5c81c772fdea36879d1be18dcbcd1926e"
+
+  option "without-docs", "Build without man pages (which requires python-sphinx)"
+  option "without-dbd-mysql", "Build without bundled Perl DBD::mysql module, to use the database of your choice."
+
+  depends_on "cmake" => :build
+  depends_on "libev" => :build
+  depends_on "sphinx-doc" => :build if build.with? "docs"
+  depends_on :mysql => :build if build.with? "dbd-mysql"
```

Comment:
I'd insist on trying to keep the bottle since it takes a considerable amount of time and build-only dependencies to build from sources rather than the ultra-fast bottle installation.

@mikemcquaid the thing is that without some form of mysql client installed, the `DBD::mysql` module build will fail (because it creates a test database as part of its build tests).

@DomT4 I think the way it is right now could work for most people, what do you say if we give it a try, and I'll make the necessary changes if there are any issues reported?